source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -52,6 +52,14 @@ private string DetectUser()
         }
 
         [AcceptVerbs(HttpVerbs.Get)]
+        [CommonJobsAuthorize(Roles = "EmployeeManagers")]
+        public ActionResult ReportDashboard()
+        {
+            //TODO: Needs UI. Will call the API for the list (GetEvaluatedEmployees)
```

Comment:
Oh, you are right, I forgot that

On Mon, Jul 27, 2015 at 12:18 PM, Gkolocsar <notifications@github.com>
wrote:

> In
> source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs
> <https://github.com/CommonJobs/CommonJobs/pull/447#discussion_r35547619>:
>
> > @@ -52,6 +52,14 @@ private string DetectUser()
> >          }
> >
> >          [AcceptVerbs(HttpVerbs.Get)]
> > +        [CommonJobsAuthorize(Roles = "EmployeeManagers")]
> > +        public ActionResult ReportDashboard()
> > +        {
> > +            //TODO: Needs UI. Will call the API for the list (GetEvaluatedEmployees)
>
> Actually that call is async and depends on the link (Ver evaluaciones).
> That's for when the EmployeeManager sees a finished evaluation and wants to
> see the auto & company califications.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CommonJobs/CommonJobs/pull/447/files#r35547619>.
>